python/lsst/ts/wep/task/combineZernikesMeanTask.py

```diff
@@ -31,5 +32,13 @@ class CombineZernikesMeanTask(CombineZernikesBaseTask):
     measurement with an unweighted mean.
     """
 
-    def combineZernikes(self, zernikeArray: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
-        return np.mean(zernikeArray, axis=0), np.zeros(len(zernikeArray), dtype=int)
+    def _combineZernikes(self, zkTable: Table) -> Table:
+        # Set all donuts to used (no rejection in this simple mean)
+        zkTable["used"] = True
+
+        # Calculate mean of every Zernike column
+        for colName in zkTable.colnames:
+            if colName.startswith("Z"):
```

Comment:
Are we ready to use `zkTable.meta["nollIndices"]` here instead of `startswith` filtering?

Reply:
Yeah I wrote that as a catch-all for all Zernike-type columns. But It's probably better to explicitly loop over all of those columns.